product_multiple_barcodes/models/product_barcode_multi.py

```diff
@@ -1,12 +1,24 @@
 # Copyright 2019 VentorTech OU
 # Part of Ventor modules. See LICENSE file for full copyright and licensing details.
 
-from odoo import models, fields
+from odoo import models, fields, api
+from odoo.exceptions import ValidationError
 
 
 class ProductBarcodeMulti(models.Model):
     _name = 'product.barcode.multi'
     _description = 'Product Barcode Multi'
 
     name = fields.Char('Barcode')
-    product_id = fields.Many2one('product.product', string='Product')
+    product_id = fields.Many2one(
+        'product.product', 
+        string='Product', 
+        required=True,
+        ondelete="cascade",
+    )
+
+    @api.constrains('name')
+    def _check_empty_name(self):
```

Comment:
Why need this and not just make name - required = True?

Reply:
Code was improved